code/pilotfile/pilotfile_convert.h

```diff
@@ -163,6 +163,8 @@ struct plr_data {
 
 	// variables
 	SCP_vector<sexp_variable> variables;
+
+	SCP_vector<sexp_container> containers;
```

Comment:
Maybe this field isn't needed? Isn't this file just for converting from the old pilot file format?

Reply:
Err, maybe I was confused by the file name `pilotfile_convert.h` that this code is for pilot file conversion?

Reply:
Could be. I probably just copied the variable vector and then never realised it wasn't needed since old files wouldn't have containers.